Filter/ModelFilter.php

```diff
@@ -100,9 +100,11 @@ protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $da
 
             $or->add($queryBuilder->expr()->notIn($alias, ':'.$parameterName));
 
-            $or->add($queryBuilder->expr()->isNull(
-                sprintf('IDENTITY(%s.%s)', $this->getParentAlias($queryBuilder, $alias), $this->getFieldName())
-            ));
+            if (ClassMetadataInfo::MANY_TO_MANY !== $this->getOption('mapping_type')) {
```

Comment:
No, it's about writing the condition with non-Yoda style:

`$this->getOption('mapping_type') !== ClassMetadataInfo::MANY_TO_MANY`

Yoda is "useful" when you risk an unwanted assignment. There is no risk here.